sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeTableExec.scala

```diff
@@ -68,6 +69,16 @@ case class DescribeTableExec(
         case (key, value) => key + "=" + value
       }.mkString("[", ",", "]")
     rows += toCatalystRow("Table Properties", properties, "")
+
+    // If any columns have default values, append them to the result.
+    if (table.schema.fields.exists(_.metadata.contains(CURRENT_DEFAULT_COLUMN_METADATA_KEY))) {
```

Comment:
This seesm duplicated with https://github.com/apache/spark/pull/39726/files#diff-632efff0ffceb2ff47b97f39570b4f2cff24401f132f559158acaa651b27af26R651. Shall we abstract them?

Reply:
Good idea, done!